File: c11_svg/B_29_Julian_Stoev.rb
def gime_a_circle x,y,r, color
	'<circle cx="'+x.to_s+'" cy="'+y.to_s+'" r="'+r.to_s+'" stroke="black" stroke-width="2" fill="'+color.to_s+'" />'
end

def rectange w,h ##missing x and y
  '<rect width="'+w.to_s+'" height="'+h.to_s+'" style="fill:rgb(255,255,0);stroke-width:5;stroke:rgb(0,0,0)"/>'
end

File.open("result.svg","w") do |f|
	f.write('<svg xmlns="http://www.w3.org/2000/svg">')
	f.write(rectange 75,150)
	f.write(gime_a_circle 35,25,25, "red")
	f.write(gime_a_circle 35,80,25, "yellow")
	f.write(gime_a_circle 35,130,25, "green")
	f.write('</svg>')
end

File: c21/B_29_Julian_Stoev.js
var array = [[1, 2, 3], [4, 5, 6], [7, 8, 9]];


var count = 0;
var i = 0;
for (count; i<array.length; count++){
    for(i; i < array[count].length; i++){
        rect(count*40, i*40, 40, 40);
    }
}
